static/app/views/settings/projectPerformance/projectPerformance.tsx

```diff
@@ -179,15 +179,6 @@ class ProjectPerformance extends AsyncView<Props, State> {
         step: 0.01,
         defaultValue: 0,
       },
-      {
-        name: 'n_plus_one_db_issue_rate',
-        type: 'range',
-        label: t('N+1 (DB) Issue Rate'),
-        min: 0.0,
-        max: 1.0,
-        step: 0.01,
-        defaultValue: 0,
-      },
```

Comment:
@mjq-sentry Might want to seperate this out to another PR to decouple frontend and backend

Reply:
D'oh, of course. Good catch, will do 👍

Reply:
https://github.com/getsentry/sentry/pull/45383